benchmarks/bench_lof.py

```diff
@@ -101,7 +97,7 @@
     fit_time = time() - tstart
     tstart = time()
 
-    scoring = -model.decision_function(X_test)  # the lower, the more normal
+    scoring = -model._decision_function(X_test)  # the lower, the more normal
```

Comment:
there is no public function that allows you to do the same?

Reply:
Hm no... actually it is maybe a pity to have a benchmark for LOF that needs to predict on a test set...
I can change this benchmark to outlier detection where we train on all the dataset (without knowing the labels) and compute the ROC on this same training set (with the knowledge of the labels). This will only require LOF scores on the training set, i.e. `negative_outlier_factor_`